feature/gnmi/ate_tests/telemetry_basic_check_test/metadata.textproto

```diff
@@ -8,12 +8,26 @@ testbed: TESTBED_DUT_ATE_2LINKS
 platform_exceptions: {
   platform: {
     vendor: CISCO
+    hardware_model: "CISCO-8202-32FH-M"
```

Comment:
hardware_model has been deprecated. 

You can use hardware_model_regex here. 

I think for this section you could use hardware_model_regex: "820.*|xrd" and in the next section you could use hardware_model_regex: "CISCO-8808" so that they are mutually exclusive.

Reply:
Thanks I was wondering why it's throwing an error all the sudden. Fixed and pushed.